### DISCUSSION TITLE
missing repro / build src for vmlinuz-3.16.0-rc4 and http://boot.ipxe.org/demo/

### DISCUSSION THREAD

**Discussion by woa7** [ipxe/ipxe#399] [Q&A]
where can i find the the build src for vmlinuz-3.16.0-rc4 and/or the full http://boot.ipxe.org/demo/ content?

i like to open a PR that:
- adding qrtext and/or qrencode for the CI test framework like https://github.com/ipxe/wimboot/blob/master/test/testwimboot
- add i386 kernel

**Comment by mcb30** [comment]
The demo kernel+initrd was built about 7 years ago using the now-long-obsolete https://github.com/mcb30/mkbootfs to create the initramfs.  It should probably be replaced by something like Tiny Core Linux.
